hono-worker/CLAUDE.md

```
@@ -0,0 +1,524 @@
+# Hono Worker Development Guide
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)